app/views/step_by_step_pages/publish_or_delete.html.erb

```diff
@@ -0,0 +1,52 @@
+<%
+  links = [
+    {
+      text: 'Step by step publisher',
+      href: step_by_step_pages_path
+    },
+    {
+      text: @step_by_step_page.title,
+      href: step_by_step_page_path(@step_by_step_page)
+    },
+    {
+      text: 'Publish or delete'
+    }
+  ]
+%>
+
+<%= render 'shared/steps/step_breadcrumb', links: links %>
+
+<%= render 'shared/steps/form_notice', message: notice, status: "success" %>
+
+<%= render 'shared/steps/heading',
+  step_nav: @step_by_step_page.title,
+  action: 'Publish or delete'
+%>
+
+<%= render 'nav', step_by_step_page: @step_by_step_page, active:"publish-or-delete" %>
+
+<div class="row publish-or-delete">
+  <div class="col-md-12">
+    <% if @step_by_step_page.has_draft? %>
+        <div class="panel panel-default">
```

Comment:
Again I think the indentation here is inconsistent - looks like it starts a few lines above with two spaces and suddenly switches to 4. Give me a shout when we're both in the office and we'll check your editor's settings.